README.md

```diff
@@ -17,21 +17,17 @@ If you use homebrew, install Docker for Mac with: `brew tap caskroom/cask && bre
 
 2) [Docker Compose](https://docs.docker.com/compose/)
 
-If you installed Docker for Mac, `docker-compose` is also installed. Otherwise install with: `pip install docker-compose`.
-
-3) [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/installing.html)
-
-Install with: `pip install awscli`. Set the `$AWS_ACCESS_KEY_ID` and `$AWS_SECRET_ACCESS_KEY` and `$AWS_REGION` environment variables. You should have permissions to evaluate the `aws ecr get-login` comamnd for your ECR region and push access to your ECR repositories.
+If you installed Docker for Mac, `docker-compose` is also installed. Otherwise install with: `(sudo) pip install docker-compose`.
 
 ## Setup
```

Comment:
Thanks @bhoggard good point - Install Hokusai, Install Dependencies (`deps` as we determined yesterday) and `setup` for apps